content/pt-br/docs/tasks/debug/debug-aplication/debug-init-containers.md

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
I believe this page belongs to your other pull request. Could you please remove it from this branch?